src/app/components/site/Homepage.tsx

```diff
@@ -71,6 +71,70 @@ export const Homepage = () => {
           </section>
         )}
 
+        {!user?.loggedIn && (
+          <section id="resources" className="resources-section">
+            <Container className="pt-4 pb-5">
+              <div className="pt-5">
+                <h2 className="homepage-sub-title text-left mb-4">Resources</h2>
+              </div>
+              <Row>
+                <div className="columns-container">
+                  <Col className="custom-col-60">
+                    <div className="resources-background">
```

Comment:
This could be a component called `Resources` which is then agnostic of it's content and scales based on it's column sizing. That would then work for the same code you have on line 101 for the question finder.

Reply:
I have added a resource component in the elements folder. 
I am now using this component for both the computer science topics and question finder.

Works really well dynamically.